### ISSUE TITLE
Fix presentation of reviews on dashboard

### ISSUE THREAD

**Issue by eshnil** [learn-awesome/learn#143]
We can take a cue from how GoodReads does it:

![image](https://user-images.githubusercontent.com/51651338/73688833-cf14c280-46c4-11ea-8d82-83b85b0cffd4.png)

**Comment by Mingos1** [comment]
Hi there, may I work on this issue?

**Comment by nileshtrivedi** [comment]
@Mingos1 Hey, that would be great! Do you want to come to the #meta channel on out Slack for discussing and finalizing the design before you start coding it?

**Comment by Mingos1** [comment]
Yes, for sure

**Comment by karanysingh** [comment]
Is this still open, can i work on this?

**Comment by nileshtrivedi** [comment]
@karanysingh Yes. Can you ping me in LearnAwesome slack group? Here is where you can join: https://learnawesome.org/join_slack

**Comment by Arkadipta-Chatterjee** [comment]
Hi..May i work on this issue?

**Comment by nileshtrivedi** [comment]
@Arkadipta-Chatterjee Since, this is a UI issue, why don't you propose a mock-up first?

**Comment by himanshu17-2000** [comment]
i want to work on this issue , please assign it to me

**Comment by nileshtrivedi** [comment]
@himanshu17-2000 We are not developing this project any more. I encourage you to explore the new version which is purely in Javascript here: https://github.com/learn-awesome/learndb
